governance/contributor-ladder.md

```diff
@@ -0,0 +1,111 @@
+If you have a project idea that aligns with the objectives of the TAG, consider writing a proposal, in a GitHub issue, outlining its motivation, scope, goals, and deliverables.
+Present this proposal during a TAG meeting to initiate a discussion and gather feedback and support from fellow members.
+Taking on a [project leadership role](roles.md#role-of-project-leads) within the community project becomes your next significant step towards leadership recognition within the group.
+
+* Take ownership of a project within the TAG
+* Lead a community group within the TAG
+
+## Helping out
+
+In open source, newcomers join the community seeking guidance and support as they embark on their journey, just as you once did.
+While it is essential to provide support to current members and leaders, it is equally important to focus on assisting and orienting newcomers.
+Modeling empathy and offering support to the community as a whole should be at the forefront of your actions.
+It is crucial to embody the values of the community as outlined in the [code of conduct](http://cncf.io/conduct/).
+
+There are various ways you can contribute to supporting and empowering new contributors.
+Firstly, you can actively participate by stepping in and offering mentorship and guidance when someone requires assistance. This could involve helping them navigate the community, providing technical guidance, or answering their questions.
+Additionally, if a member is unable to fulfill their responsibilities, you can offer to fill in their role temporarily or assist in any necessary tasks.
+Co-hosting meetups and events is another way to provide support. By sharing the responsibility, you help create an inclusive environment where everyone feels supported and encouraged.
+
+Lastly, building a habit of consistently helping out within the TAG is highly valuable.
+This can include engaging in discussions, providing feedback on proposals, reviewing code contributions, or offering assistance in any way that aligns with your skills and expertise.
+By actively participating and contributing, you demonstrate your commitment to the community and showcase the values that the Environmental Sustainability TAG holds dear.
+
+* Helping out and mentor new contributors in their open source journey.
+* Helping out community members and leaders if they need support.
+* Demonstrating empathy and embody the community values.
+
+## Leading a Working Group
+
+Within the Environmental Sustainability TAG, various ongoing initiatives are carried out through dedicated working groups.
+As leadership roles naturally transition over time, it is worthwhile for you to consider taking up a leadership position within an established working group.
+Alternatively, you can contribute by proposing the formation of a new working group.
+To do so, create a GitHub issue where you outline the motivation, scope, goals, and deliverables of the proposed working group.
```

Comment:
I'd probably swap this bullet and the one after.  I usually have discussion to see if I have sufficient backing before making a new group/issue/initiative.

Reply:
Changed the section to this.. 😅 

> Since leadership roles naturally change over time, it is worth your while to take a leadership position in an established working group. Within the TAG for Environmental Sustainability, dedicated working groups carry out several ongoing initiatives. However, if the current working groups do not provide space for the topic you are interested in, you could start a new working group. A working group must have community support and requires that you find sufficient backing before a working group is formed. To start this process, create a GitHub issue describing the initial charter in which you outline the motivation, scope, goals, and deliverables of the proposed working group. This working group charter document should form the basis for discussions with the TAG before initiating an official vote by the TAG co-chairs, which also requires approval by the CNCF TOC. It is important to present the working group proposal during TAG meetings and be vocal about the discussions, actively solicit feedback and input, and gain support from other members.